client/components/Pipeline/Kanban.tsx

```diff
@@ -178,13 +180,22 @@ export default class Kanban extends PureComponent<KanbanProps, KanbanState> {
       <div>
         <h2>{`${this.props.user.firstName} ${this.props.user.lastName}`}</h2>
         <div className="pipelineButtons">
-          <CustomDropdown partners={this.state.partnerNames} />
+          <PartnerDropdown partners={this.state.partnerNames} />
           <IndividualButton
             loggedInPartnerName={`${this.props.user.firstName} ${
               this.props.user.lastName
             }`}
           />
           <GroupButton />
+          <PartnerTeamDropdown partnerTeams={Object.keys(Team)} />
+          {/* <IndividualButton
```

Comment:
I should delete the commented out stuff here

Reply:
Yea can you delete